### DISCUSSION TITLE
couple SiPM to ccintillator crytal with liquid glass/liquor silicium

### DISCUSSION THREAD

**Discussion by AF1977** [OpenGammaProject/Open-Gamma-Detector#36] [Ideas]
Hi,

What about to use liquid glass/liquor silicium instead of clear silikon grease? May more clear than silicon grease and works as glue.

https://en.wikipedia.org/wiki/Sodium_silicate

br

Andreas

**Comment by NuclearPhoenixx** [comment]
Hi, no idea to be honest.

It needs to have a refractive index of about 1.5, it should be completely transparent even after hardening, it must not expand or contract when hardening and it shouldn't chemically attack any of the surfaces, i.e. the scintillator window glass or the SiPM coating.

If it checks all these boxes, you can try. YMMV + I hope I didn't forget anything.

At that point, though, I'd just buy one of the million optically clear adhesives out there.

**Comment by sebyon** [comment]
Hey Andreas, 

I would personally avoid using sodium silicate based products, mostly as the temperature and pretreatment of chemicals required will most likely damage the SiPM. I would suggest reading the Onsemi guide on handling of the SiPM to understand what conditions it can and cannot handle. https://www.onsemi.com/download/application-notes/pdf/and9788-d.pdf

A high viscoity silicon grease also serves two other purposes other than RI matching. It's high viscoisty actually helps the SiPM adhere to the crystal, and allows servicability in case you wish to change or upgrade the SiPM, by being able to be cleaned off and reapplied. I would also suggest reading this. https://www.logwell.com/tech/servtips/optical_coupling_grease.html

There are commerical permenant options if you so wish to go down that route. Norland do optical adhesives with various RI, so you can match to the exact RI of your crystal choice. The RI of NaI(Ti) is actually closer to 1.85, and it is easier to find adheasives near this value.

**Comment by AF1977** [comment]
Hi guys,

Thanks for quick answers. I read the manuals from onsemi. And with sodium silicate it will only work on J-SiPM series. They covered with glass.
Do you have a link for opticale grade grease for me that works in Germany or Europe?

Thanks in advance.

BR 

Andreas

**Reply by sebyon** [reply]
Hey Andreas, 

I'm not 100% sure for Europe/Germany. A nice hack I learned was to look for Silicon Differential Oil for RC cars, as they come in small volumes and are essentially rebottled Dow Corning 200 . They will specify the cSt. If you see one that is clear and 100,000 cSt, you're in.

**Comment by dc1rdb** [comment]
This is what I am using with excellent results:
![IMG_20230612_062159](https://github.com/OpenGammaProject/Open-Gamma-Detector/assets/9314230/3bf464f6-9eb6-46c2-b75d-6684ceb7a3c6)
It is normally being used for servicing coffee machines :-)

**Reply by NuclearPhoenixx** [reply]
That's exactly what I'm using. Cheap and effective. It's just not an adhesive, that's the only downside if that's what you're looking for.

**Comment by AF1977** [comment]
Hi guys,

Thanks for the replies. I found this:

https://www.siliconesandmore.com/de/silikonoel-100000-cst.html?id=215466052&gclid=CjwKCAjw4ZWkBhA4EiwAVJXwqZMn0qmvegMp-Ngl5UIe-ltHUzrhiOoVeTlPvcKfyci53L-GC-YO0xoCWJMQAvD_BwE

    Viskosität: 100.000 cSt
    Zündung Temperatur > 300°C
    Gefrierpunkt: -50°C
    Dichte bei 25°C: 0.97
    Oberflächenspannung: 21.5 mN/m
    Brechungsindex bei 25°C: 1.404

RI looks good 1.404, viscosity is 100000 cSt and clear color.

BR 

Andreas

**Comment by NuclearPhoenixx** [comment]
Closing this as it looks like it's resolved. If you have any further questions, please don't hesitate!
